schema/accepted_schema.sql

```diff
@@ -830,7 +830,7 @@ CREATE INDEX IF NOT EXISTS "idx_ElectionVotes_electionName" ON "ElectionVotes" U
 -- Table "ElicitQuestionPredictions"
 CREATE TABLE "ElicitQuestionPredictions" (
   _id VARCHAR(27) PRIMARY KEY,
-  "prediction" DOUBLE PRECISION NOT NULL,
+  "prediction" DOUBLE PRECISION,
```

Comment:
There has already been a migration for this in https://github.com/ForumMagnum/ForumMagnum/commit/185d8af5fe491abefe4457c0401ca23bc226f986